docs/modeling_services.md

```diff
@@ -0,0 +1,38 @@
+# Modeling Services in Kubernetes
+
+Generally speaking there are two ways to model [Services](http://kubernetes.io/v1.0/docs/user-guide/services.html) in Kubernetes:
+
+1. Service providers (i.e. a database) define their own services
+2. Service consumers (i.e. an application) define the services they consume
+
+With the first option, service consumers must constantly track the service providers they depend on.  When a service provider changes, all consumers must be notified and restarted.  This results in tight coupling between services.
```

Comment:
Rather than: "This results in tight coupling between the services." you may want to say: "This results in tight coupling between the service consumers and suppliers."  (The same for the next line also.)